client/jetpack-cloud/sections/agency-dashboard/sites-overview/sites-dataviews/index.tsx

```diff
@@ -192,7 +263,21 @@ const SitesDataViews = ( { data, isLoading, onViewChange }: SitesDataViewsProps
 					}
 					return item.site.value.blog_id;
 				} }
-				onChangeView={ setView }
+				onChangeView={ (
+					newView: SetStateAction< {
+						type: string;
+						perPage: number;
+						page: number;
+						sort: { field: string; direction: string };
+						search: string;
+						filters: { field: string; operator: string; value: number }[];
+						hiddenFields: string[];
+						layout: object;
+						selectedSite: undefined;
```

Comment:
In the PR I'm working on, (PreviewPaneLayout) I created these interfaces:

```
export interface Sort {
	field: string;
	direction: 'asc' | 'desc';
}

export interface Filter {
	field: string;
	operator: string;
	value: string;
}

export interface SitesViewState {
	type: 'table' | 'list' | 'grid';
	perPage: number;
	page: number;
	sort: Sort;
	search: string;
	filters: Filter[];
	hiddenFields: string[];
	layout: object;
	selectedSite?: Site | undefined;
}
```

Reply:
Works well for me now that I've implemented the changes in 87646. However I did change the filter value to be a number as that is what it was expecting (with my implementation anyway...)